docs/master/eloquent/nested-mutations.md

```diff
@@ -476,38 +476,68 @@ input UpdateAuthorInput {
 
 ## MorphTo
 
+__The GraphQL Specification does not support Input Union types,
+for now we are limiting this implementation to `connect` and `disconnect` operations.__
+
 ```graphql
+type Task {
+  id: ID
+  name: String
+}
+
+type Hour {
+  id: ID
+  weekday: Int
+  hourable: Task
+}
+
 type Mutation {
   createHour(input: CreateHourInput! @spread): Hour @create
+  updateHour(input: UpdateHourInput! @spread): Hour @update
 }
 
 input CreateHourInput {
-  hourable_type: String!
-  hourable_id: Int!
   from: String
   to: String
   weekday: Int
+  hourable: CreateHourableOperations
 }
 
-type Hour {
-  id: ID
+input UpdateHourInput {
+  id: ID!
+  from: String
+  to: String
   weekday: Int
-  hourable: Task
+  hourable: UpdateHourableOperations
 }
 
-type Task {
-  id: ID
-  name: String
-  hour: Hour
+input CreateHourableOperations {
+  connect: ConnectHourableInput
+}
+
+input UpdateHourableOperations {
+  connect: ConnectHourableInput
+  disconnect: Boolean
+}
+
+input ConnectHourableInput {
+  type: String!
+  id: ID!
 }
 ```
 
+You can use `connect` to associate existing models.
+
 ```graphql
 mutation {
   createHour(input: {
-    hourable_type: "App\\\Task"
-    hourable_id: 1
     weekday: 2
+    hourable: {
+      connect: {
+        type: "Tests\\\Utils\\\Models\\\Task"
```

Comment:
Can we shorten this type to just `Task`?

Reply:
The Laravel default is to use the full namespace, so let's go with `App\\\Models\\\Task`